src/App.tsx

```diff
@@ -1,26 +1,63 @@
 import React from 'react';
 import logo from './logo.svg';
+import axios from 'axios';
 import './App.css';
+import { usePAPforms } from './usePAPforms';
+import { UserForm } from './UserForm';
+import { useState } from 'react';
+import { FormEvent } from 'react';
+
+type FormData = {
+  filePath: string
+  password: string
+}
+
+const INITIAL_DATA: FormData= {
+  filePath: "",
+  password: ""
+}
 
 function App() {
```

Comment:
We should have routing to show this form only on localhost:PORTNUM/settings

take a look at: https://www.w3schools.com/react/react_router.asp